cpp/include/raft/cluster/detail/kmeans_common.cuh

```diff
@@ -369,6 +370,9 @@ void minClusterAndDistanceCompute(
                           true,
                           stream);
   } else {
+    // TODO: Unless pool allocator is used, passing in a workspace for this
+    // isn't really increasing performance because this needs to do a re-allocation
+    // anyways.
```

Comment:
Not sure if there is a github issue, but the todo seems unclear, the todo is to remove the re-allocation or to remove passing workspace? Either way, a GH issue link would be useful

Reply:
The todo is to remove the workspace from the public APIs. We do have an issue for it but let me reference that in the code for completeness.